klasses/api.py

```diff
@@ -210,3 +212,144 @@ def fetch_bootcamp_run(run_property):
                     run_filters
                 )
             ) from exc
+
+
+def create_run_enrollment(user, run, order=None):
+    """
+    Creates local records of a user's enrollment in bootcamp runs, and attempts to enroll them
+    in novoed via API
+
+    Args:
+        user (User): The user to enroll
+        run (BootcampRun): The bootcamp run to enroll in
+        order (ecommerce.models.Order or None): The order associated with these enrollments
+
+    Returns:
+        (BootcampRunEnrollment): enrollment object that were successfully created
+
+    """
+    enrollment, _ = BootcampRunEnrollment.objects.update_or_create(
+        user=user,
+        bootcamp_run=run,
+        defaults={"active": True, "change_status": None},
+    )
+
+    try:
+        enrollment.user.profile.can_skip_application_steps = True
+        enrollment.user.profile.save()
```

Comment:
I can't trust it while running the management command it would be either username or id but not the user instance so that's why preferring to use the `enrollment.user.profile`